libgadget/blackhole.c

```diff
@@ -320,7 +322,7 @@ blackhole(ForceTree * tree)
                     (0.1 * LIGHTCGS * LIGHTCGS * THOMPSON)) * All.UnitTime_in_s);
 
         fprintf(FdBlackHoles, "%g %d %g %g %g %g\n",
-                All.Time, SlotsManager->info[5].size, total_mass_holes, total_mdot, mdot_in_msun_per_year, total_mdoteddington);
+                All.Time, total_bh, total_mass_holes, total_mdot, mdot_in_msun_per_year, total_mdoteddington);
```

Comment:
It would be great if we can add a file header every now and then  -- (numpy access # comment lines). A file with just numbers is a bit confusing.. But it may choke other downstream parsers.

Reply:
hah, yes, a good idea. I don't know of any downstream parsers for these files